slsReceiverSoftware/include/receiver_defs.h

```diff
@@ -36,7 +36,7 @@
 
 //parameters to calculate fifo depth
 #define SAMPLE_TIME_IN_NS				100000000//100ms
-
+#define MAX_EIGER_ROWS_PER_READOUT		(256)
```

Comment:
Are the () needed? Also maybe we can run format on this file, seems like the alignment is off...

Reply:
It doesn't need it and again had looked fine with the editor, but will do the formatting for all the files at a later step.

Reply:
done with branckets